themes/qtail/assets/main.css

```diff
@@ -11,6 +11,10 @@
         font-style: normal;
         font-display: swap;
     }
+
+    body {
+        overflow-x: hidden;
+    }
```

Comment:
isn't it simpler to shrink the offending overflowing element? if someone inadvertently adds some stuff that does overflow, it would be nice to be able to see it.

Reply:
Do you really think that the website will ever have horizontal scrolling? overflow-x disables horizontal scrolling only within the body. This won't affect things like code block windows, etc.

Reply:
I guess my thought is: some bug is causing horizontal scrolling, why not fix that bug? Disabling horizontal scrolling means that if someone introduces it (as you say, probably accidentally), users won't be able to scroll until someone comes along and fixes it.

Right now that element is probably invisible, so it's not an issue. But some day it may not be.

Reply:
i think the problem is the margin inline of "current release" parent element. 

![2023-11-13-181615_1366x768_scrot](https://github.com/qtile/qtile.org/assets/53843982/f735e5b1-3485-4e0f-beec-b91c1e929754)

I think that decreasing margin would be a quickfix.

Reply:
I agree, can you send a patch? :)

Reply:
sure!